ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template2.kt

```diff
@@ -115,10 +127,43 @@ private fun SelectPackageButton(
         shape = RoundedCornerShape(15.dp),
         border = border,
     ) {
-        Text(
-            text = "Purchase ${packageInfo.rcPackage.identifier}. " +
-                "Price: ${packageInfo.rcPackage.product.price.formatted}",
-        )
+        Column(
+            modifier = Modifier.fillMaxSize().padding(4.dp),
+            verticalArrangement = Arrangement.spacedBy(4.dp),
+        ) {
+            Row(horizontalArrangement = Arrangement.spacedBy(6.dp), verticalAlignment = Alignment.CenterVertically) {
+                CheckmarkBox(isSelected = isSelected, colors = state.currentColors)
+                Text(
+                    text = packageInfo.localization.offerName ?: packageInfo.rcPackage.product.title,
```

Comment:
I see you matched the iOS behavior here. I don't remember this logic but I guess it makes sense hehe